patches/tModLoader/Terraria/Terraria.csproj.patch

```diff
@@ -0,0 +1,13 @@
+--- src/Terraria/Terraria/Terraria.csproj
++++ src/tModLoader/Terraria/Terraria.csproj
+@@ -82,10 +_,6 @@
+     <PropertyGroup Condition="$(DefineConstants.Contains('DEBUG'))">
+         <EditBinOpts>$(EditBinOpts) /stack:0x800000</EditBinOpts>
+     </PropertyGroup>
+-    <Message Text="Applying editbin $(EditBinOpts)" Importance="high" />
+-    <Exec Command="&quot;$(SolutionDir)largeaddresstool\editbin&quot; $(EditBinOpts) &quot;$(TargetPath)&quot;" />
+-	<Message Text="Checking headers with dumpbin." Importance="high" />
+-	<Exec Command="&quot;$(SolutionDir)largeaddresstool\dumpbin&quot; $(DumpBin) &quot;$(TargetPath)&quot; | findstr &quot;large&quot;" />
```

Comment:
Please don't get rid of this.

Reply:
Sure, I had only removed it to get it to compile